databases/week3/databases_week3.sql

```diff
@@ -17,6 +17,9 @@ id INT UNSIGNED NOT NULL AUTO_INCREMENT,
 number_of_guests INT UNSIGNED NOT NULL,
 meal_id INT UNSIGNED NOT NULL,
 created_date DATETIME NOT NULL DEFAULT NOW(),
+contact_phonenumber VARCHAR(20) NOT NULL,
```

Comment:
Now you should also add these fields in the queries that insert data into the reservation, otherwise they crash ;)